### PR TITLE
Unskip CI after holidays

### LINKED ISSUE #638: Revert PR 637 on Jan 3rd
This PR disabled CI runs during the holidays, Revert on Jan 3rd when the oss-bot coiled account is enabled back.
- https://github.com/coiled/coiled-runtime/pull/637

### PR DESCRIPTION
**Pull request by hendrikmakait**
* Closes #638 
* Reverts #637

### CODE REVIEW

**Review by ntabris — APPROVED** [review]

**Comment by hendrikmakait** [comment]
@ntabris: Are CI errors expected or should they already be fixed?

https://cloud.coiled.io/dask-engineering/clusters/137662/software/2828

**Comment by ntabris** [comment]
> Are CI errors expected or should they already be fixed?
> 
> https://cloud.coiled.io/dask-engineering/clusters/137662/software/2828

@shughes-uk you know what's happening here? The software logs show that senv was built and uploaded by `18:10:22UTC`, the worker logs show that worker got 424 and gave up at `18:11:19UTC`.

**Comment by hendrikmakait** [comment]
Matching CI logs: https://github.com/coiled/coiled-runtime/actions/runs/3831858841/jobs/6521500308#step:5:2658

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ requirements:
   run:
     - python >=3.8,<3.11
     - pip
-    - coiled >=0.2.54
+    - coiled >=0.2.58
```
Comment (hendrikmakait) [inline recipe/meta.yaml:25]: Looking at the CI failures in https://github.com/coiled/coiled-runtime/actions/runs/3831858841/jobs/6521497028 and other jobs in the same run, should we update `coiled` to `0.2.58`?

Comment (shughes-uk) [reply recipe/meta.yaml:25]: 0.2.58 is already installed by `>=0.2.54` , this is a race condition I have a fix for

**Pull request merged**